src/arrayMethodReduce.js

```diff
@@ -5,7 +5,28 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator;
+    let startIndex;
```

Comment:
You can declare them with correct values and just change them if arguments.length is > 2. This will help to avoid extra checks